### PR TITLE
UI design

### PR DESCRIPTION
**Pull request by EleonoraGalieva**
Images of main pages are made in figma (link in readme). Though this pr has some css in it, please don't pay attention to it for now.

### CODE REVIEW

**Comment by gmltA** [comment]
I don't see any HTML tags on your figma muckups.

**Review by gmltA — CHANGES_REQUESTED** [review]
Can you please make your PR follow the [acceptance criteria](https://github.com/gmltA/FrontEnd_tasks#acceptance-criteria)? I need only images in it's content.

**Review by gmltA — COMMENTED** [review]
I'm messed up. What is this PR for? 2 or 3 milestone?

**Comment by EleonoraGalieva** [comment]
This pr is for 3 milestone(ui design).

**Comment by EleonoraGalieva** [comment]
I've deleted js files from this pr so it follows the acceptance criteria of milestone 3 (ui design).

**Comment by gmltA** [comment]
Where's the deployment link?

**Comment by EleonoraGalieva** [comment]
Added deployment link: https://eleonoragalieva.github.io/MyWallet

File: categories.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            <li class="category">
+                <p>Food</p>
+                <img class="category__img" src="img/food_category.png" alt="Food category image">
+            </li>
+            <li class="category">
+                <p>Fun</p>
+                <img class="category__img" src="img/fun_category.png" alt="Fun category image">
+            </li>
+            <li class="category">
+                <p>Groceries</p>
```
Comment (gmltA) [inline categories.html:58]: image

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,1076 @@
...
+    text-align: start;
+}
+
+ul {
+    padding: 0;
+}
+
+:focus {
+    outline: 0;
+}
```
Comment (gmltA) [inline css/styles.css:95]: Well, you have to come up with own focus styles then.

**Review by gmltA — COMMENTED** [review]
That's a <kbd>6</kbd> for now. Add `:hover` and `:focus` styles to interactive elements.

**Comment by EleonoraGalieva** [comment]
What else can I fix? I also have a theme-change, but it uses some js, which is in diffrenet branch, so it is not showing fully on this one (https://github.com/EleonoraGalieva/MyWallet/blob/Front-end-basics/js/theme-change.js).

**Comment by gmltA** [comment]
Add `focus` styles to interactive elements to improve the mark. They are still missing.

File: css/styles.css [hunk 3]
```diff
@@ -385,6 +426,11 @@ a {
     padding: 5px;
 }
 
+.input:focus {
+    outline: none;
+    box-shadow: 0 0 4px 2px var(--lightOrange);
+}
+
```
Comment (EleonoraGalieva) [inline css/styles.css:449]: I also have some :focus on input in all forms.

**Review by gmltA — APPROVED** [review]
Okay, that's a <kbd>9</kbd>.

**Comment by EleonoraGalieva** [comment]
Great!)

**Pull request merged**